src/dot/commons/video/video_utils.py

```diff
@@ -6,13 +6,73 @@
 
 import os
 import random
-from typing import Callable, Dict
+from typing import Callable, Dict, Union
 
 import cv2
+import mediapipe as mp
 import numpy as np
+from mediapipe.python.solutions.drawing_utils import _normalized_to_pixel_coordinates
 from tqdm import tqdm
 
-from ..utils import find_files_from_path
+from ..pose.head_pose import pose_estimation
+from ..utils import expand_bbox, find_files_from_path
+
+mp_face = mp.solutions.face_detection.FaceDetection(
+    model_selection=0,  # model selection
+    min_detection_confidence=0.5,  # confidence threshold
+)
```

Comment:
comment for later. we should find a better solution than global initialisation here.